netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
> I know that, but I am not sure if we need this added feature right now :)

IMHO it is needed because without API versioning this is your best way to compare the netctl/netmaster compatibility.